internal/test/protobuf/test.proto

```diff
@@ -19,4 +19,5 @@ option (gogoproto.testgen_all) = true;
 
 message TestMessage {
     string message = 1;
+    int32 duration = 2;
```

Comment:
Can we limit scope of this PR to only zerolog? And left any logic changes to another PR ?

Reply:
Ah, it is caused by #158. nevermind